rest/replication_api_test.go

```diff
@@ -541,9 +540,7 @@ func TestReplicationRebalancePull(t *testing.T) {
 	defer base.SetUpTestLogging(base.LevelInfo, base.KeyReplicate, base.KeyHTTP, base.KeyHTTPResp, base.KeySync, base.KeySyncMsg)()
 
 	// Disable sequence batching for multi-RT tests (pending CBG-1000)
-	oldFrequency := db.MaxSequenceIncrFrequency
-	defer func() { db.MaxSequenceIncrFrequency = oldFrequency }()
-	db.MaxSequenceIncrFrequency = 0 * time.Millisecond
+	db.SuspendSequenceBatching()
```

Comment:
This isn't calling the returned function to reset to original state.

```suggestion
	defer db.SuspendSequenceBatching()()
```